app/javascript/packs/dashboardTagsDisableUnchangedButtons.js

```diff
@@ -1,35 +1,36 @@
-document.getElementById('follows_update_form').addEventListener('submit', checkChanged);
```

Comment:
Should this file be in here? It mostly seems like formatting changes, no actual code changes? Probably better to do this in a separate cleanup PR.